packages/builtin/string.pony

```diff
@@ -209,7 +209,7 @@ actor Main
   fun box _copy_to(ptr: Pointer[U8] ref, copy_len: USize,
     from_offset: USize = 0, to_offset: USize = 0) =>
     """
-    Copy copy_len characters from this to that at specified offsets.
+    Copy copy_len single-byte characters from this to that at specified offsets.
```

Comment:
We should just say "Copy copy_len bytes from"... since characters don't always map to a single byte depending on the encoding.

Reply:
updating